content/security/reporting.adoc

```diff
@@ -96,6 +96,16 @@ The following behaviors/issues are not vulnerabilities in Jenkins project infras
 * The public availability of the Algolia API key. It needs to be public.
 // TODO Possibly other keys as well?
 
+==== External web applications
+
+The Jenkins project manages only the web applications hosted under the `*.jenkins.io` domain.
```

Comment:
Possibly `jenkins-ci.org`, plus the problem that someone may find the IP address and not know the host name -- what then?